Snowflake.Data.Tests/IntegrationTests/SFDbDataReaderIT.cs

```diff
@@ -872,32 +844,29 @@ public void testGetChars()
 
                 reader.Close();
 
-                cmd.CommandText = "drop table if exists testGetChars";
-                count = cmd.ExecuteNonQuery();
-                Assert.AreEqual(0, count);
-
                 conn.Close();
             }
         }
 
         [Test]
         public void testGetStream()
         {
+            CreateOrReplaceTable(TableName, new []
```

Comment:
what connection does it use? Previously it was using the connection provided within the test and I'm wondering if we should not refactor and call this with conn param within using clause? wdyt?

Reply:
Good idea, now we can call the method with already created connection.